src/AsyncDemoScreen.js

```diff
@@ -0,0 +1,82 @@
+import { useState } from 'react'
+import { StatusBar } from 'expo-status-bar';
+import { StyleSheet, Text, View, Button, TextInput, TouchableOpacity } from 'react-native';
+import { setStringValue, getMyStringValue, getAllKeys, removeValue, clearAll } from './utils/asyncStorage';
+
+/** Async Demo Screen. */
+const CreateMScreen = () => {
```

Comment:
assuming this isnt a real panel? do we want to merge this?

Reply:
Ill move to another dir